scripts/protocgen.sh

```diff
@@ -2,9 +2,26 @@
 
 set -eo pipefail
 
+protoc_gen_gocosmos() {
+  if ! grep "github.com/gogo/protobuf => github.com/regen-network/protobuf" go.mod &>/dev/null ; then
+    echo -e "\tPlease run this command from somewhere inside the regen-ledger folder."
+    return 1
+  fi
+
+  go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos 2>/dev/null
+}
+
+protoc_gen_doc() {
+  go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc 2>/dev/null
+} 
+
+protoc_gen_gocosmos
+protoc_gen_doc
+go mod tidy
```

Comment:
This is a bit unfortunate: `protoc_gen_doc` modifies go.mod/sum, so I need to tidy it after to have no diff.